starlite/asgi/routing_trie/mapping.py

```diff
@@ -54,8 +54,10 @@ def add_map_route_to_trie(
                 if component.type is Path:
                     current_node["is_path_type"] = True
                     break
-
-                next_node_key: Union[Type[PathParameterSentinel], str] = PathParameterSentinel
+                # if component.type in current_node["children"]:
+                #     raise ImproperlyConfiguredException("Cannot have adjacent path parameter nodes of the same type.")
```

Comment:
I think this would be correct behavior, but if enabled there are a lot of test errors.